news/354.update.rst

```diff
@@ -0,0 +1,2 @@
+Update ``cv2`` hook to collect extra config files and modules for compatibility with OpenCV 4.5.4.58.
```

Comment:
4.5.4.58 -> 4.5.4.60, since 4.5.4.58 still does not work due to module switching attempt.

Reply:
Sure, changed it to `4.5.4.60`. Just thought that it would work with `pyinstaller`, because python loader was added to OpenCV python package in `4.5.4.58` version.